pysindy/optimizers/sbr.py

```diff
@@ -76,22 +76,27 @@ def __init__(
         slab_shape_nu=4,
         slab_shape_s=2,
         noise_hyper_lambda=1,
-        normalize_columns=False,
-        copy_X=True,
-        **mcmc_kwargs,
+        num_warmup=1000,
+        num_samples=5000,
+        mcmc_kwargs=None,
+        **kwargs,
     ):
-        super().__init__(
-            copy_X=copy_X,
-            normalize_columns=normalize_columns,
-        )
+        super().__init__(**kwargs)
```

Comment:
Since optimizers all inherit the `unbais` kwarg and `_unbias()` step from `BaseOptimizer`, and that step appears incompatible with SBR, I think we should explicitly add `unbias: bool=False` to the init args here, guard `if unbias: raise ValueError` with a message like [this one](https://github.com/dynamicslab/pysindy/blob/a7eb8901f1e333273ef8ec17ad6dab61db15c2ab/pysindy/optimizers/trapping_sr3.py#L326), and pass `unbias` explicitly in the superclass constructor`.

Reply:
Okay dokey. This is fixed in https://github.com/dynamicslab/pysindy/pull/440/commits/28dd0f3ac1675c31f732d0c0a7791b3305107c28